docs/configuration.asciidoc

```diff
@@ -88,8 +88,8 @@ Here is a sample `appsettings.json` configuration file for a typical ASP.NET Cor
 <1> With ASP.NET Core, you must set `LogLevel` for the internal APM logger in the standard `Logging` section with the `Elastic.Apm` category name.
 <2> The configurations below `ElasticApm` are fetched by the agent if the corresponding `IConfiguration` is passed to the agent.
 
-In certain scenarios--like when you're not using ASP.NET Core--you won't activate the agent with the `UseElasticApm()` method.
-In this case, you can set the agent's log level with <<config-log-level,`ElasticApm:LogLevel`>>, as shown in the following `appsettings.json` file:
+In certain scenarios--you won't activate the agent with the `UseElasticApm()` method.
```

Comment:
stray `--` still present- maybe replace with comma?

Reply:
Ahh, dang, I hadn't refreshed the page so this comment didn't show up until after I pressed the merge button. Let me quick open a new PR to get this fixed 🙃

Reply:
I'm not sure why the "like when you're not using ASP.NET Core" was removed here tbh. I think it makes sense to keep it.